Makefile

```diff
@@ -175,7 +174,7 @@ override deps = \
 	$(refs)
 
 $(addprefix bin/, $(deps)):
-	$(CD) dep/debugger-libs && $(NUGET) restore debugger-libs.sln && $(MSBUILD) $(MSBUILD_FLAGS) debugger-libs.sln
+	$(CD) dep/debugger-libs && $(MSBUILD) $(MSBUILD_FLAGS) '/t:Restore;Build' debugger-libs.sln
```

Comment:
Do we need the explicit `Restore` invocation? I thought these days it will be invoked automatically by `Build`?

Reply:
You could also just specify `/r` so `Build` invokes restore, it doesn't by default.

Reply:
Replaced with `/r`, which is shorter and sounds nicer for me, with commit 7051d8834511a6bc944b196703f7e0c7030a69d1.